modules/UI/UI.js

```diff
@@ -188,13 +188,6 @@ UI.changeDisplayName = function(id, displayName) {
  */
 UI.setRaisedHandStatus = (id, name, raisedHandStatus) => {
     VideoLayout.setRaisedHandStatus(id, raisedHandStatus);
```

Comment:
I haven't read your entire changeset but knowing your style you probably made the raise hand indicator tied into redux so it hides and shows itself. Would that mean then UI.setRaisedHandStatus could be removed as long as the video thumbnail ahs the raised hand indicator component?

Reply:
Very good point! Added an additional commit to do that. Thanks!